### PR TITLE
Update the main README and add two README for hostPath and NFS manifests.

### PR DESCRIPTION
**Pull request by LaContra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,135 @@
...
-* [kind](https://kind.sigs.k8s.io/)
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Install a k8s distribution and build a cluster.
+- [Install Helm](https://helm.sh/docs/intro/install/) on the k8s controller node.
+
+### Install and configure NFS
+
+1. Set up NFS (Example: [Guide for NFS installation and configuration on Ubuntu](https://ubuntu.com/server/docs/service-nfs)).
+
+2. Create beacon node and validator data folders with the correct ownership (we use uid 1001 and gid 2000) on NFS.
```
Comment (chingyi-lin) [inline README.md]: maybe "(Our helm chart uses uid 1001 and gid 2000 by default)"?

File: README.md [hunk 2]
```diff
@@ -1,51 +1,135 @@
...
     ```
 
-2. Create the data folders for beacon node and validator on the host machine. For example:
-    ```
-    mkdir -p /data/prysm/validator /data/prysm/beacon
-    ```
+2. Change values in `./eth2prysm/values.yaml`.
 
-3. Update the `extraMounts` in `cluster-config/kind-single-node.yaml` with the path to the created data folder.
+   It's recommended to read through each field in `values.yaml` and determine the desired values. Fields that need to be changed or verified before installing the chart are the following ones:
```
Comment (chingyi-lin) [inline README.md]: `It's recommended to read through` -> We recommend checking each field in `values.yaml` to determine the desired configuration.

File: README.md [hunk 3]
```diff
@@ -1,51 +1,135 @@
...
 
-4. Create the validator wallet and replace `example-password` with your wallet password in `host-path/wallet-secret.yaml`.
+   - **nfs.serverIp**: NFS server IP address.
+   - **image.version**: Prysm client version.
+   - **beacon.dataVolumePath**: The beacon node data volume path/folder on NFS.
```
Comment (chingyi-lin) [inline README.md]: The path to the data directory on the NFS for beacon node.

File: README.md [hunk 4]
```diff
@@ -1,51 +1,135 @@
...
+
+- Check the status of the first validator (To check other validators, change -lapp to other validators' names).
+
+   ```bash
+   kubectl logs -f -nprysm -lapp=validator1
+   ```
+
+## For Development or Testing
+
+If you want to develop for this project or verify your configuration quickly, we recommend the following setup:
```
Comment (chingyi-lin) [inline README.md]: ... quickly without setting up NFS or other storage solution,

File: README.md (hunk 1 above)
Comment (LaContra) [reply README.md]: Changed.

File: README.md (hunk 2 above)
Comment (LaContra) [reply README.md]: Changed.

File: README.md (hunk 3 above)
Comment (LaContra) [reply README.md]: Changed. Also changed the descriptions for `validators.validator1.dataVolumePath` and `validators.validator1.walletVolumePath`.

File: README.md (hunk 4 above)
Comment (LaContra) [reply README.md]: Added.

**Review by chingyi-lin — APPROVED** [review]

**Pull request merged**